dhis-2/dhis-services/dhis-service-tracker/src/test/java/org/hisp/dhis/tracker/validation/hooks/EventDataValuesValidationHookTest.java

```diff
@@ -117,22 +118,6 @@ public void successValidationWhenCreatedAtIsNull()
         assertThat( reporter.getReportList(), hasSize( 0 ) );
     }
 
-    @Test
```

Comment:
Were these tests deleted because the behaviour changed or they are not relevant anymore here?
In the first case, we should just change them to reflect the change, in the second case, we should just move them were appropriate.

Reply:
@enricocolasante 
yes more or less, the point is that fields of type Instant, now cannot be invalid, since he serializer will fail before even reaching the validation layer. This behaviour is already tested at EventImportValidationTest:testWrongScheduledDateString where it is checked that proper exception is thrown. I don't think we need to repeat same kind of test for all fields, however if needed I might add another specific one for this

Reply:
I am just confused because here the test was asserting that there were no error if the data was invalid, while now we are failing if the date is invalid.

I think that if a created or updated date was invalid, we were just ignoring them.
Are we ok with this change?